client/src/featureform/resources.py

```diff
@@ -373,6 +385,8 @@ class Source:
     variant: str = "default"
     schedule: str = ""
     schedule_obj: Schedule = None
+    is_transformation = 0
```

Comment:
Is there a reason why we are using `0` and `1` versus `False` and `True`?

Reply:
Yes, some versions of SLQite3 dont support true and false